.github/workflows/lint-and-typecheck.yml

```
@@ -0,0 +1,87 @@
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Run Lint
+        run: pnpm lint
+
+  type-check:
+    runs-on: ubuntu-latest
+    needs: [setup, lint]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.3)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)